pupil-spa/test/README.md

```diff
@@ -0,0 +1,98 @@
+Installing and Running MTC tests
+================================
+
+This is a short guide to installing and running the MTC tests.  
+
+##Clone the tests
+
+```bash
+git clone https://agilefactory.visualstudio.com/DefaultCollection/MTC/_git/app
+```
+
+##Install RVM stable with ruby
+
+Follow installation instructions for installing RVM here:
+
+https://rvm.io/rvm/install
+ 
+Once rvm is installed, we need ruby version 2.4.0, to install use the following:
+ `rvm install 2.4.0`
+ 
+Use ruby version 2.4.0 and set it as your default:
+ `rvm use 2.4.0 --default`
+
+##Installing required gems
+
+Go to the folder /test and run the following commands:
+
+`gem install bundler` and then `bundle install`
+
+These two commands will install all the gems that the tests need.
+
+## Running the tests
+
+#####Basic
+
+To run the tests with the default options ( headless and app running on localhost:3000 ) give the command below:
+
+`cucumber`
+ 
```

Comment:
which directory do we run cucumber from? i assume `test` but best to be explicit.